### DISCUSSION TITLE
Manage thumbnails in the GUI ->proposal

### DISCUSSION THREAD

**Discussion by Bellavista** [blakeblackshear/frigate#14254] [Ask A Question]
Hi  @blakeblackshear , @NickM-27 ,  @hawkeye217
Thanks for your work!
I'm looking for some settings to manage thumbnails in the GUI, I would like the thumbnails to be always visible.
I've read some requests on this topic more or - similar to mine but not closed.
I would like to propose you to solve the problem using the already existing variable "**Mark as reviewed**" linked to thumbnails.
The proposal is to insert some a variable in the ui: configuration section as showed low:
```# Optional: Configuration for how camera is handled in the GUI.
    ui:
      # Optional: Adjust sort order of cameras in the UI. Larger numbers come later (default: shown below)
      # By default the cameras are sorted alphabetically.
      order: 0
      # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
      dashboard: True
      --- proposal to add -----
      # Optional: Configuration for video thumbnails in the GUI.
      thumbnails:
      # True -  the thumbnails are present available retain days 
      # False - the thumbnails are not present 
      # Auto - the thumbnails are present available until reviewed and view available for retain days (default: shown below)
         prevideo: Auto
          retain:
            # Required: Default retention days (default: shown below)
            default: 10
            # Optional: Per object retention days
            objects:
              person: 15
      --- proposal to add -----
```

for each new detection -> snapshots if prevideo variabel is:
- True -> the "**Mark as Reviewed**" is On, not modifible on GUI available until retain days
- False -> the "**Mark as Reviewed**" is Off, the thumbnails are not visible
- Auto-> the "**Mark as Reviewed**" is On, modifible on GUI available until retain days  (managed like today)

i hope to be clear
Best Regard
@bellavista

check if this discussion are linked 
https://github.com/blakeblackshear/frigate/discussions/13824 -> Unanswered
https://github.com/blakeblackshear/frigate/discussions/13530 -> Generale on goin 
https://github.com/blakeblackshear/frigate/discussions/12049 -> Generale on goin 
https://github.com/blakeblackshear/frigate/discussions/12816 -> Config Support an inprued

**Comment by NickM-27** [comment]
It's not clear to me what you are suggesting. Why would mark as reviewed be forced on or off? Even if you want the alert thumbnails to be visible that really has nothing to do with the mark as reviewed button.

**Reply by Bellavista** [reply]
i try to expaly better ( probality i inverted on and off in proposal before :-(...  )

in this image i suppose the two thumbnails "mark as reviewd" are OFF and  they are visible in GUI
![image](https://github.com/user-attachments/assets/89380318-c443-4649-850c-a3744e773a2d)
in Review i can see the person simbols grey "OFF"
![image](https://github.com/user-attachments/assets/71de68b7-430a-4dab-8490-ee9a5fc939c2)
if i see the thumbnails the scenario is changed ...
in this image i suppose the two thumbnails "mark as reviewd" are changed ON and  they are not visible in GUI
![image](https://github.com/user-attachments/assets/50894a75-dbe4-4654-9b89-7af456a5494b)
in Review i can see the person simbols are green "ON"
![image](https://github.com/user-attachments/assets/9ccddd19-c875-4623-83ce-a48574538c87)

from what I have described the thumbnail view are linked by the "Mark as review" state.

1 - I would always like the "Mark as review" always OFF to have the thumbnails always visible.
2 - Some would like to have "Mark as review" always ON because they do not want the thumbnails.
3 - today the system is a mix of all the solutions.

for this reason, I suggested a "prevideo" variable
1- True 2- False 3 - Auto to manage all this

Best Regard

**Reply by NickM-27** [reply]
Why do you want to see images of items that you have already reviewed? The advantage of hiding them is that you can quickly see if anything important has occurred.

**Reply by Bellavista** [reply]
nick-27, good question!
My son and I manage the home automation. (cliar!....my wife is the boss ;-).
I could find more answers to the question.
1- I might want this option.
2- If another user sees the videos and you will not see them.
3 - similar to point 1, versatility to satisfy the various choices of different users

**Reply by NickM-27** [reply]
Gotcha, I don't think that is a good way to solve that issue. Frigate already supports authentication, so we can use that to have marking items as reviewed to be user specific instead of global
